docs/topics/what-is-the-toolkit.adoc

```diff
@@ -11,14 +13,21 @@
 [discrete]
 == What is the {ProductName}?
 
-The {ProductName} ({ProductShortName}) is an extensible and customizable rule-based tool that simplifies the migration of Java applications.
+The {ProductName} ({ProductShortName}) is an extensible and customizable rule-based tool that simplifies the migration and modernization of Java applications.
 
-The {ProductShortName} examines application artifacts, including project source directories and application archives, then produces an HTML report that highlights areas needing changes. {ProductShortName} can migrate Java applications from earlier versions of Red Hat JBoss Enterprise Application Platform or from other application servers, such as Oracle WebLogic Server or IBM WebSphere Application Server.
+{ProductShortName} examines application artifacts, including project source directories and application archives, then produces an HTML report that highlights areas needing changes. {ProductShortName} supports many migration paths including:
+
+* Application servers to Red Hat JBoss Enterprise Application Platform
+* Application containerization and cloud readiness
+* Spring Boot to Quarkus
+* Updates to OpenJDK
+
+For more information see, link:https://developers.redhat.com/products/mta/use-cases[Migration Toolkit for Applications use cases and migration paths].
 
 [discrete]
 == How does the {ProductName} simplify migration?
 
-The {ProductName} looks for common resources and highlights technologies and known trouble spots when migrating applications. The goal is to provide a high-level view into the technologies used by the application and provide a detailed report organizations can use to estimate, document, and migrate enterprise applications to Java EE and Red Hat JBoss Enterprise Application Platform.
+The {ProductName} looks for common resources and highlights technologies and known trouble spots when migrating applications. It provides a high-level view of the technologies used by the application as well as a detailed report evaluating a proposed migration or modernization. Using this report, an organization can estimate the efforts required for large-scale modernization and migration projects and take specific steps to reduce the work involved.

```

Comment:
Paragraph breaks would be helpful.

Suggestion:

The {ProductName} looks for common resources and known trouble spots when migrating applications. It provides a high-level view of the technologies used by the application.

{MTA} generates a detailed report evaluating a migration or modernization path. This report can help you to estimate the effort required for large-scale projects and to reduce the work involved.